cue/model/api/v1/dashboard_go_gen.cue

```diff
@@ -4,7 +4,7 @@
 
 package v1
 
-import "github.com/perses/perses/cue/model/api/v1/common"
+import "github.com/perses/perses/pkg/model/api/v1/common"
```

Comment:
this change shouldn't exist. Are you on a Mac ? If it's the case I guess the `sed` command didn't work.

Reply:
I am, and I did see some sed errors, I thought it was fishy. I'll fix that.